### DISCUSSION TITLE
How to change font?

### DISCUSSION THREAD

**Discussion by adojck** [dailydotdev/daily#228] [General]
Hello,
Is there a way to change the font for "New Tab"? In "Layout Settings" there is no such option. Or maybe there is a way to set font to be not mono-spaced? I don't enjoy the appearance of text at the moment.

![image](https://user-images.githubusercontent.com/4030692/102361097-21946000-3fb3-11eb-87fd-83cb1f0548b6.png)

**Comment by idoshamun** [comment]
Hey @adojck,

Unfortunately, it's not possible to change the font at the moment. We're in the process of creating a new typography guidelines that will use the default system font. It will optimize readability and performance.
It should take a while though...
